djpyfs/djpyfs.py

```diff
@@ -98,12 +91,19 @@ def get_osfs(namespace):
     if not os.path.exists(full_path):
         os.makedirs(full_path)
     osfs = OSFS(full_path)
-    osfs = patch_fs(osfs, namespace, lambda self, filename, timeout=0:os.path.join(djfs_settings['url_root'], namespace, filename))
+    osfs = patch_fs(osfs, namespace, lambda self, filename, timeout=0: os.path.join(djfs_settings['url_root'], namespace, filename))
     return osfs
 
 def get_s3fs(namespace):
     ''' Helper method to get_filesystem for a file system on S3 '''
-    global key_id, key_secret
+    # These imports are temporary to get to a place where we can test while not incurring a boto dependency... yet.
```

Comment:
This comment needs a few more words to be clear. It took a few minutes to figure out. Perhaps: 

"Our test suite does not presume Amazon S3, and we would prefer not to have a global import so that we can run tests without requiring boto. These will become global when and if we include S3/boto in our test suite."